sale_order_delivery_date/models/sale_order.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    delivery_expected_date = fields.Date("Expected Delivery Date")
```

Comment:
```suggestion
    dispatch_expected_date = fields.Date(
        "Expected Dispatch Date",
        states={'draft': [('readonly', False)], 'sent': [('readonly', False)]},
        copy=False,
        readonly=True,
        help="The input date will be passed over to the delivery as Scheduled Date."
    )
```
To apply the consistent design as commitment_date.